src/main/java/racingcar/domain/RaceRecords.java

```diff
@@ -0,0 +1,49 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class RaceRecords {
+
+    private final List<RaceRecord> raceRecords;
+
+    public RaceRecords() {
+        this.raceRecords = new ArrayList<>();
+    }
+
+    public List<RaceRecord> getRaceRecords() {
+        return raceRecords;
+    }
+
+    public Map<String, Integer> saveRecords(Cars cars) {
```

Comment:
RaceRecord 를 반환해보는건 어떨까요? 😄 
```suggestion
    public RaceRecord saveRecords(Cars cars) {
```